src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.builds

```diff
@@ -2,21 +2,16 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="runtime.native.System.IO.Compression.pkgproj"/>
```

Comment:
Chatted with @chcosta and he says we need them so will add them back.  I still think there's a bug here though with respect to their content (absence thereof).